src/test/integration/users/User.t.sol

```diff
@@ -72,16 +72,111 @@ contract User is PrintUtils {
         return _NAME;
     }
 
-    /*******************************************************************************
-                            DELEGATIONMANAGER METHODS
-    *******************************************************************************/
+    /// -----------------------------------------------------------------------
+    /// Allocation Manager Methods
+    /// -----------------------------------------------------------------------
+    
+    /// @dev Allocates randomly accross the operator set's strategies with a sum of `magnitudeSum`.
+    /// NOTE: Calling more than once will lead to deallocations...
+    function modifyAllocations(OperatorSet memory operatorSet, uint64[] memory magnitudes) public virtual createSnapshot {
+        print.method(
+            "modifyAllocations",
+            string.concat(
+                "{avs: ",
+                Logger(operatorSet.avs).NAME_COLORED(),
+                ", operatorSetId: ",
+                cheats.toString(operatorSet.id),
+                "}"
+            )
+        );
+
+        IStrategy[] memory strategies = allocationManager.getStrategiesInOperatorSet(operatorSet);
+
+        require(strategies.length == magnitudes.length, "User.modifyAllocations: length mismatch");
+
+        AllocateParams[] memory params = AllocateParams({
+            operatorSet: operatorSet,
+            strategies: strategies,
+            newMagnitudes: magnitudes
+        }).toArray();
+
+        allocationManager.modifyAllocations(params);
+
+        rollForward({blocks: allocationManager.ALLOCATION_CONFIGURATION_DELAY()});
```

Comment:
We only need to roll forward past the operator's allocation delay, not the allocation configuration delay.

Reply:
oops fixed in [80d79e8](https://github.com/Layr-Labs/eigenlayer-contracts/pull/894/commits/80d79e83ac657137fd10a1559ae385bf08d5660f)